testing/src/main/java/com/baeldung/lambdabehave/Calculator.java

```diff
@@ -0,0 +1,27 @@
+package com.baeldung.lambdabehave;
+
+public class Calculator {
+
+    private int x;
+    private int y;
+
+    Calculator(int x, int y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    public int add() {
+        return this.x + this.y;
+    }
+
+    public int divide(int a, int b) {
+        if (b == 0) {
```

Comment:
No need for that `if` block. ArithmeticException is throw automatically when the division by zero happens